src/Oscoin/Data/OscoinTx.hs

```diff
@@ -283,6 +284,17 @@ insertAccount acc@Account{..} = WorldState.insert (accountKey accountId) (Accoun
 deleteAccount :: CBOR.Serialise (PublicKey c) => AccountId c -> WorldState c -> WorldState c
 deleteAccount id = WorldState.delete (accountKey id)
 
+-- | Transfer balance between two accounts. Uses 'creditAccount' and 'debitAccount'.
+transferBalance
+    :: (CBOR.Serialise (PublicKey c))
+    => AccountId c
+    -> AccountId c
```

Comment:
Could you document which account is the sender and which is recipient to avoid misuse?

Reply:
That's part of the :game_die: :trollface:

Reply:
8edca46